packages/edit-site/src/components/actions/index.js

```diff
@@ -27,7 +27,7 @@ const { useHistory } = unlock( routerPrivateApis );
 export const trashPostAction = {
 	id: 'move-to-trash',
 	label: __( 'Move to Trash' ),
-	isPrimary: true,
+	isPrimary: false,
```

Comment:
This is handled by default when not `true`, so we can remove this line completely. Additionally we can remove the `icon` too, as it's used only for primary actions.